.prettierignore

```diff
@@ -0,0 +1,3 @@
+/node_modules
+/coverage
+/build
```

Comment:
node_modules is covered automatically

coverage and build are in .gitignore so no need to prettierignore

if you were to include these, you do not need the starting slash (`node_modules` over `/node_modules`), you want to start with relative paths

I'd say it's pretty rare that .prettierignore is actually needed, in this case it can be removed